packages/codemod/src/upgrade-legacy.ts

```diff
@@ -103,6 +102,42 @@ const upgradeLegacy = async () => {
       )
       parsedProgram.value.program.body.unshift(importWithBlitz)
       config.remove()
+      
+      const findImports = program.find(j.ImportDeclaration, (node) => node)
+      if (findImports.length) {
+        findImports.forEach((i) => {
+          const defaultImport = i.value.specifiers?.find((s) => s.type === "ImportDefaultSpecifier")
+          const namedImports = i.value.specifiers?.filter((s) => s.type === "ImportSpecifier")
+          let flag=false
+          if (defaultImport && defaultImport.local) {
+            const importStatement = j.expressionStatement(
+              j.assignmentExpression(
+                "=",
+                j.identifier("const " + defaultImport.local.name),
+                j.callExpression(j.identifier("require"), [j.stringLiteral(i.value.source.value as string)]),
```

Comment:
```suggestion
                j.callExpression(j.identifier("require"), [j.stringLiteral(`${i.value.source.value})]),
```

We're trying to avoid assertions as much as possible. This would be safer in my opinion.